INSTALL.md

```diff
@@ -21,18 +21,18 @@ git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
 cd /path/to/cgal.git
 mkdir -p build/debug
 cd build/debug
-cmake -DBUILD_TYPE=Debug ../..
+cmake -DCMAKE_BUILD_TYPE=Debug ../..
 make
 ```
 
 Here is an example of how to build the library in Release:
 ``` {.bash}
-  > git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
-  > cd /path/to/cgal.git
-  > mkdir -p build/debug
-  > cd build/debug
-  > cmake -DBUILD_TYPE=Release ../..
-  > make
+git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
+cd /path/to/cgal.git
```

Comment:
I'm not sure including the `.git` suffix in the example directory name makes a ton of sense. I believe that's only used for [bare repos](http://git-scm.com/book/ch4-2.html).